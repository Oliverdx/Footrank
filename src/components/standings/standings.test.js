import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';

import Standings from './standings';

describe('Tests of the Standings Component', () => {

    it('Prevent the component to not render if not receive props', () => {
        const component = renderer
            .create(<Standings />)
            .toJSON();

        expect(component).toMatchSnapshot();
    });
});
