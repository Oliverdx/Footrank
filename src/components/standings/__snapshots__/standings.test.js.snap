// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests of the Standings Component Prevent the component to not render if not receive props 1`] = `null`;
